actors/miner/src/lib.rs

```diff
@@ -4184,17 +4185,25 @@ fn request_terminate_deals(
     deal_ids: Vec<DealID>,
 ) -> Result<(), ActorError> {
     const MAX_LENGTH: usize = 8192;
-
     for chunk in deal_ids.chunks(MAX_LENGTH) {
-        extract_send_result(rt.send_simple(
+        let res = extract_send_result(rt.send_simple(
             &STORAGE_MARKET_ACTOR_ADDR,
             ext::market::ON_MINER_SECTORS_TERMINATE_METHOD,
             IpldBlock::serialize_cbor(&ext::market::OnMinerSectorsTerminateParamsRef {
                 epoch,
                 deal_ids: chunk,
             })?,
             TokenAmount::zero(),
-        ))?;
+        ));
+        // Intentionally swallow this error to prevent frozen market cron corruption from also freezing this miner cron.
+        // This is safe from a malicious caller perspective because this method's callstack should always originate with the trusted system caller.
```

Comment:
Please update this comment; it's confusing, given the code below that denies the "should".